assets/variant-list.js

```diff
@@ -154,6 +154,9 @@ class VariantList extends HTMLElement {
       sections_url: window.location.pathname
     });
```

Comment:
Some of the elements are moving once the error appears

https://screenshot.click/30-54-xbyxe-x6svz.mp4